SAAdMobBannerAd.swift

```diff
@@ -0,0 +1,78 @@
+//
+//  SAAdMobBannerAd.swift
+//  SuperAwesomeAdMob
+//
+//  Created by Gunhan Sancar on 19/01/2023.
+//
+
+import GoogleMobileAds
+import SuperAwesome
+
+@objc(SAAdMobBannerAd)
+class SAAdMobBannerAd: NSObject, GADMediationBannerAd {
+    var view: UIView {
+        return bannerAd ?? UIView()
+    }
+    
+    required override init() {
+        super.init()
+    }
+    
+    /// AwesomeAds banner ad.
+    var bannerAd: BannerView?
+    
+    /// The ad event delegate to forward ad rendering events to the Google Mobile Ads SDK.
+    var delegate: GADMediationBannerAdEventDelegate?
+    
+    /// Completion handler called after ad load
+    var completionHandler: GADMediationBannerLoadCompletionHandler?
+    
+    func loadBanner(
+        for adConfiguration: GADMediationBannerAdConfiguration,
+        completionHandler: @escaping GADMediationBannerLoadCompletionHandler
+    ) {
+        let parameter = adConfiguration.credentials.settings["parameter"] as? String ?? ""
+        let placementId = Int(parameter) ?? 0
+        let adSize = adConfiguration.adSize
+        
+        bannerAd = BannerView(frame: CGRect(x: 0, y: 0, width: adSize.size.width, height: adSize.size.height))
```

Comment:
```suggestion
        bannerAd = BannerView(frame: CGRect(x: 0.0, y: 0.0, width: adSize.size.width, height: adSize.size.height))
```

One of the things to fractionally improve compile times is to get in to the habit of declaring float/doubles with .0

Reply:
Nice one